src/test/analyzer_test.ts

```diff
@@ -12,15 +12,17 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-'use strict';
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
```

Comment:
I'm honestly not sure it's better to take it out, I was more curious why some projects didn't need it. Considering I'd rather just import it, it's probably better to leave it. Importing doesn't work, does it?